.github/workflows/deploy-docs.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Documentation'
+name: 'Documentation (Push Event)'
```

Comment:
I don't like the naming together with the PR workflow name. It is to long imho as it is shown concatenated with the job name. Maybe we use "Documentation" and "PR Docs" or keep both at "Documentation"

Reply:
Ah, no worries. This was due to when I had two separate workflows for tags and push events, as they both showed up in the Actions tab as "Documentation" which wasn't that helpful while developing it. The additional context was useful imo, but feel free to change it :+1: